mitiq/calibration/calibrator.py

```diff
@@ -96,95 +83,96 @@ def _get_performance(
+        │ Ideal distribution: {'00': 0.5, '11': 0.5} │ Factory: RichardsonFactory     │ Noisy error: 0.0157     │
+        │ Num qubits: 2                              │ Scale factors: [1.0, 2.0, 3.0] │ Mitigated error: 0.0018 │
+        │ Circuit depth: 2                           │ Scale method: fold_global      │                         │
+        │ Two qubit gate count: 1                    │                                │                         │
+        ├────────────────────────────────────────────┼────────────────────────────────┼─────────────────────────┤
+        │ Type: ghz                                  │ Technique: ZNE                 │ ✔                       │
+        │ Ideal distribution: {'00': 0.5, '11': 0.5} │ Factory: RichardsonFactory     │ Noisy error: 0.0157     │
+        │ Num qubits: 2                              │ Scale factors: [1.0, 3.0, 5.0] │ Mitigated error: 0.0091 │
+        │ Circuit depth: 2                           │ Scale method: fold_global      │                         │
+        │ Two qubit gate count: 1                    │                                │                         │
+        └────────────────────────────────────────────┴────────────────────────────────┴─────────────────────────┘
+        """  # noqa: E501
+        table: List[List[str | float]] = []
+        headers: List[str] = ["benchmark", "strategy", "performance"]
+        for problem in self.problems:
+            row_group: List[List[str | float]] = []
+            for strategy in self.strategies:
+                perf, nerr, merr = self._get_performance(
                     strategy.id, problem.id
                 )
-                yield problem, strategy, performance_symbol, nerr, merr
-
-    def log_technique(self, technique: MitigationTechnique) -> str:
-        """Creates a table displaying all results of a given mitigation
-        technique."""
-        table: List[List[Union[str, float]]] = []
-        for (
-            problem,
-            strategy,
-            performance_symbol,
-            noisy_error,
-            mitigated_error,
-        ) in self._technique_results(technique):
-            row: List[Union[str, float]] = [
-                performance_symbol,
-                problem.type,
-                technique.name,
-            ]
-            summary_dict = strategy.to_pretty_dict()
-            if strategy.technique is MitigationTechnique.ZNE:
-                row.extend(
+                row_group.append(
                     [
-                        summary_dict["factory"],
-                        summary_dict["scale_factors"],
-                        summary_dict["scale_method"],
+                        str(problem),
+                        str(strategy),
+                        f"{perf}\nNoisy error: {round(nerr, 4)}\n"
+                        f"Mitigated error: {round(merr, 4)}",
```

Comment:
Let's add the improvement factor after these as an additional useful metric.

$$\text{improvement factor} = \left|\frac{\text{ideal} - \text{noisy}}{\text{ideal} - \text{mitigated}}\right|$$

Reply:
Good idea. Added.